### DISCUSSION TITLE
Start of a new chapter

### DISCUSSION THREAD

**Discussion by RicCodeX** [community/community#49819] [GitHub Education]
I am using this medium to introduce myself as i just signed in to github education, and want to begin my tech journey. I hope to learn, collaborate, and create.

**Comment by bbkx226** [comment]
Greetings on GitHub! I cordially invite you to follow my profile and kindly consider showing your appreciation by giving a star to my work.

**Comment by Ntshangase** [comment]
Hello, Welcome to the community, Be sure to check out the amazing opportunities at the GitHub global campus: https://education.github.com/students
